examples/browserify/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Could you use `/node_modules`? Otherwise this matches any kind of node modules folder, which might not be what you want.